### ISSUE TITLE
Texture streatch

### ISSUE THREAD

**Issue by Verdilac** [spite/THREE.DecalGeometry#7]
Is there any way to prevent decals from stretching textures? I need to create mechanism that works just like real-life decal. Can you give me any advice how to achieve it?

![streatch](https://cloud.githubusercontent.com/assets/6390981/21246387/a666ac86-c328-11e6-9bc0-aca24a76a6d4.png)

**Comment by spite** [comment]
Not really, since that's the way the decal projection works. There isn't a way implemented to solve that problem.

Maybe your shader could fade the texture when the angle between the surface and the projector origin is > 90º? that would be an option.

**Comment by spite** [comment]
Did this help?
